dockstore-common/src/main/java/io/dockstore/common/Utilities.java

```diff
@@ -85,7 +86,7 @@ public static INIConfiguration parseConfig(String configFile) {
         return executeCommand(command, true, Optional.of(ByteStreams.nullOutputStream()), Optional.of(ByteStreams.nullOutputStream()), null);
     }
 
-    public static ImmutablePair<String, String> executeCommand(String command, File workingDir) {
```

Comment:
Elaborating on the comment above, this will change it so that anything that calls `executeCommand(String command, File workingDir)` or `executeCommand(String command)` will have a timeout now. They may not have expected it in the past.

For example, https://github.com/dockstore/dockstore-cli/blob/develop/dockstore-client/src/main/java/io/dockstore/client/cli/nested/BaseLauncher.java#L112-L114 might mean that running workflows would also be under a time limit now which doesn't make sense.

I think we basically need a "opt-in" timeout rather than a default. 
Sorry for not elaborating before